src/applications/claims-status/tests/components/ClaimListItemV3.unit.spec.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
```

Comment:
We should be checking that other flags (`developmentLetterSent`, `documentsNeeded`) are not rendered when `decisonLetterSent` is true